Main.java

```diff
@@ -9,7 +10,19 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+  Scanner input = new Scanner(System.in);
+    // declare a variable that will store a name
+    int Celsius;
```

Comment:
Great work! My only feedback would be to remember variable names always begin with a lowercase letter. We use the camelCasing format. So celsius would be a better name for this variable instead of Celsius